src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,75 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Pair pair = new Pair();
```

Comment:
initialize this in the constructor
```suggestion
    private final Pair[] elements;
```